pkg/kritis/attestation/validated_attestation.go

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright 2019 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package attestation
+
+// ValidatedAttestation represents a trusted and verified attestation made by
+// named attestor about named artifact.  Any claims are contained in the
```

Comment:
Rename this to AttestationAuthority for consistency with the current state of the codebase?